<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>606</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>712</width>
    <height>606</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>712</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(230, 250, 250, 255), stop:1 rgba(255, 255, 255, 255));

 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop:0 rgba(245, 240, 235, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>229</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_dashboard">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 50px;</string>
       </property>
       <property name="text">
        <string>داشبورد</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_person">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 50px;
selection-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
       </property>
       <property name="text">
        <string>اشخاص</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_data">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 50px;</string>
       </property>
       <property name="text">
        <string>تحلیل داده</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_tafazol">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>N02Jeyran</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 50px;</string>
       </property>
       <property name="text">
        <string>مغایرت گیری</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_manage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 50px;</string>
       </property>
       <property name="text">
        <string>مدیریت بودجه ریزی</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>B Nazanin</family>
          </font>
         </property>
         <property name="text">
          <string>پشتیبانی</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>B Nazanin</family>
          </font>
         </property>
         <property name="text">
          <string>بلاگ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_call">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>B Nazanin</family>
          </font>
         </property>
         <property name="text">
          <string>تماس با ما</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>0</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>711</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>711</width>
      <height>51</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>0</y>
       <width>61</width>
       <height>49</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>B Titr</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>آباک</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_calender_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>16</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1402-07-18</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_0">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>15</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/calendar.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/profile.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/refresh.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>636</x>
       <y>13</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/logo-no-background.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-12</x>
      <y>50</y>
      <width>581</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/dash.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>442</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/icons/inno-tech.png);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>50</y>
      <width>531</width>
      <height>530</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="setting">
     <attribute name="title">
      <string>تنظیمات</string>
     </attribute>
     <widget class="QLineEdit" name="line_recdaftar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>44</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>349</x>
        <y>40</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>تعداد رکورد دفتر خود را وارد کنید:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>173</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>بازه ای  که رکوردهای شما را شامل شود را وارد کنید:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>233</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>292</x>
        <y>233</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>236</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>از</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>237</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>تا</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>293</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>برای شروع فرآیند مغایرت گیری بر روی دکمه زیر کلیک فرمایید:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_start">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>363</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 50px;
background-color: rgb(153, 255, 182);
</string>
      </property>
      <property name="text">
       <string>کلیک کنید</string>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_recbank">
      <property name="geometry">
       <rect>
        <x>211</x>
        <y>113</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>109</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>تعداد رکورد بانک خود را وارد کنید:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>خروجی</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>540</width>
        <height>510</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>510</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>510</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>37</number>
      </attribute>
      <column>
       <property name="text">
        <string>دفتر</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>بانک</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>خروجی اکسل</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>برای گرفتن خروجی اکسل برو روی آیکون پایین کلیک نمایید:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_excel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>140</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 70px;
background-color: rgb(153, 255, 182);</string>
      </property>
      <property name="text">
       <string>کلیک نمایید</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>label_4</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_5</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
